_tags

```diff
@@ -16,57 +16,55 @@ true: annot, bin_annot
 "_darcs": not_hygienic
 # Library sqlexpr
 "src/sqlexpr.cmxs": use_sqlexpr
-<src/*.ml{,i,y}>: pkg_csv
```

Comment:
Spent some time trying to figure this out, and I'm actually still not sure of the answer. In any case, ocamlfind is bounded at 1.5.2 by most versions of ppx_tools [1], so that minimum applies to sqlexpr transitively.

[1] https://github.com/ocaml/opam-repository/blob/master/packages/ppx_tools/ppx_tools.0.99.2/opam